tools/perf-automation/tests.yml

```
@@ -310,7 +310,8 @@ stages:
 - stage: Print_Results
   displayName: Print Results
   pool:
-    name: azsdk-pool-mms-ubuntu-2204-general
+    name: azsdk-pool
+    demands: ImageOverride -equals ubuntu-22.04
```

```suggestion
    demands: ImageOverride -equals ubuntu-24.04
```

Can we also bump to 24 here?